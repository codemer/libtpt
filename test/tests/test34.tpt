@tpt_ignoreblankline\
@tpt_ignoreindent\


<html>
	<head><title>My Page</title></head>

<body>
@tpt_noignoreblankline\

<h1>Title</h1>

