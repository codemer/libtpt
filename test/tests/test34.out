<html>
<head><title>My Page</title></head>
<body>

<h1>Title</h1>

